webpki-ccadb/Cargo.toml

```diff
@@ -20,3 +20,7 @@ serde = { version = "1.0.183", features = ["derive"] }
 webpki = { workspace = true }
 x509-parser = { workspace = true }
 yasna = { workspace = true }
+
+[features]
+default = []
+web = ["pki-types/web"]
```

Comment:
Do you have a usecase for building this crate for a web target? It's primarily used for codegen for the data vendored into the webpki-roots crate.